ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master-presubmits.yaml

```diff
@@ -65,6 +65,73 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/e2e-openstack-byon
+    decorate: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-api-provider-openstack-master-e2e-openstack-byon
+    optional: true
+    path_alias: sigs.k8s.io/cluster-api-provider-openstack
+    rerun_command: /test e2e-openstack-byon
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
No need for inverse regexps anymore; see `skip_if_only_changed`, e.g. #19409.

Reply:
ok